### DISCUSSION TITLE
Header e Param como JSon Object

### DISCUSSION THREAD

**Discussion by MiltonSonoda** [viniciussanchez/RESTRequest4Delphi#79] [Q&A]
Vinicius, Header e Param não poderiam aceitar um JSon Object como o Body? Ao invés de AddHeader e AddParam aceitarem somente strings?

**Comment by viniciussanchez** [comment]
Até poderia, mas não é comum esse tipo de situação. Pra falar a verdade nunca vi uma pessoa passando passando um JSON na URL e nem em um Header. O JSON foi criado para ser trafegado no body da requisição seguindo as boas práticas.

**Comment by MiltonSonoda** [comment]
Entendi. Mas o que eu queria dizer seria eu passar um JSON (Mesmo em string) e o RestRequest transformar em Key/Value internamente para trafegar. Obrigado.
